scripts/Makefile

```diff
@@ -0,0 +1,96 @@
+# to install geocint just run 
+# make install configuration_file=/path/to/your/config/file/config.inc.sh
+
+# Add here export for every varible from configuration file that you are going to use in targets
+export PGDATABASE = $(shell sed -n -e '/^PGDATABASE/p' ${configuration_file} | cut -d "=" -f 2)
+export PGUSER = $(shell sed -n -e '/^PGUSER/p' ${configuration_file} | cut -d "=" -f 2)
+export GENERAL_FOLDER = $(shell sed -n -e '/^GENERAL_FOLDER/p' ${configuration_file} | cut -d "=" -f 2)
```

Comment:
I don't see why there need to be something called `GENERAL_FOLDER` and can't find any explanation of this variable in README.

Reply:
Looks like we can remove this variable.